PROJECT_README.md

```diff
@@ -1,45 +1,45 @@
-<img align="right" width="250" height="47" src="/Material/images/Gematik_Logo_Flag.svg"/> <br/> 
-
-# < PROJECT NAME >
-
-<details>
-  <summary>Table of Contents</summary>
-  <ol>
-    <li>
-      <a href="#about-the-project">About The Project</a>
-       <ul>
-        <li><a href="#release-notes">Release Notes</a></li>
-      </ul>
-	</li>
-    <li><a href="#license">License</a></li>
-    <li><a href="#contact">Contact</a></li>
-  </ol>
-</details>
-
-## About The Project
```

Comment:
Hier würde ich schon einen Text schreiben, was man hier findet und warum es dieses Repo gibt. 
Den gleichen Text kann man auch auf der Simplifier Startseite schreiben.

Sowas angeben wie 
* Das sind die Fehlercodes des E-Rezept-Fachdienstes
* Die Fehler sind fix und werden vom Betreiber des FD eingestellt
* Und auch dein Konzept wie die Fehler funktionieren/aufgebaut sind (zumindest grob, Details in der API)

Reply:
Oder das kann in die Readme. Weiß nicht was der Unterschied zwischen README und PROJECT_README ist